### PR TITLE
Better computed docs

### PR DESCRIPTION
**Pull request by thelinuxlich**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -173,6 +173,25 @@ import { proxyWithComputed } from 'valtio/utils'
...
+  doubled: snap => snap.count * 2
+})
+```
+
+Computed values accept custom setters too:
+
+```js
+import { proxyWithComputed } from 'valtio/utils'
+
+const state = proxyWithComputed({
```
Comment (dai-shi) [inline readme.md:185]:
```suggestion

// Computed values accept custom setters too:

const state2 = proxyWithComputed({
```

Let's make it as short as possible.

File: readme.md [hunk 2]
```diff
@@ -173,6 +173,25 @@ import { proxyWithComputed } from 'valtio/utils'
...
+  firstName: 'Alec',
+  lastName: 'Baldwin'
+}, {
+  fullName: {
+    get(snap) { return snap.firstName + ' ' + snap.lastName },
+    set(newValue, state) {
+       const names = newValue.split(' ')
+       state.firstName = names[0]
+       state.lastName = names[names.length - 1]
+    }
```
Comment (dai-shi) [inline readme.md:195]:
```suggestion
    get: (snap) => snap.firstName + ' ' + snap.lastName,
    set: (state, newValue) => { [state.firstName, state.lastName] = newValue.split(' ') },
```

You might want to deal with middle name, but let's do this.

**Review by dai-shi — COMMENTED** [review]
Thanks for the suggestion. Two comments from me.

**Comment by thelinuxlich** [comment]
We are missing a async computed example but I don't know if it would belong in this section or in the React Suspense section.

**Comment by dai-shi** [comment]
We can keep it secret for now. (It's pretty experimental.)
If you would like to add it, please add it to the source code comment.
Keeping readme small is crucial for us.

**Review by dai-shi — APPROVED** [review]

**Pull request merged**